azure-function-http/src/main/java/io/micronaut/azure/function/http/AzureCookies.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.azure.function.http;
+
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.core.convert.ArgumentConversionContext;
+import io.micronaut.core.convert.ConversionService;
+import io.micronaut.http.HttpHeaders;
+import io.micronaut.http.cookie.Cookie;
+import io.micronaut.http.cookie.Cookies;
+import io.micronaut.http.netty.cookies.NettyCookie;
+import io.netty.handler.codec.http.cookie.ServerCookieDecoder;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Implementation of {@link Cookies} for serverless.
+ *
+ */
+@Internal
+public final class AzureCookies implements Cookies {
```

Comment:
I think we should move this class to `io.micronaut.netty.cookies` in core.

Reply:
@timyates create an issue for this and add it to [`5.1.0 Release`  project](https://github.com/orgs/micronaut-projects/projects/53/views/1).